### ISSUE TITLE
Speeds over 12 m/s fail to load on Mini 4 Pro

### ISSUE THREAD

**Issue by ivangayton** [hotosm/drone-tm#329]
# TL:DR

We need to limit flight plan speeds to a maximum of 12 m/s in order to fly the DJI Mini 4 Pro. If a speed above 12 m/s is specified, the controller adjusts it down to 2.5 m/s and the flight plan proceeds at far too low a speed.

## The probleme

If I create a flight plan with speeds over 11 m/and upload it to the RC2 controller attached to my Mini 4 Pro, it transfers to the drone without any problems, but when I try to load it onscreen it offers to "adjust and open" rather than simply opening. If I agree, the plan loads.

However, when I fly the drone, it proceeds at a ridiculously low speed.

It turns out that the "adjustment" when the flight plan is opened on the controller is replacing the global speed (which at my given parameters worked out to 12.65 m/s) with a default of 2.5 m/s. 

If I manually lower the speeds to 12 m/s (or lower), it opens without complaint and flies the path normally.

# proposed solution 
Cap all speeds in flight plans for the Mini 4 Pro at 12 m/s max.

**Comment by nrjadkry** [comment]
Speed is calculated based on different parameters like gsd, AGL, image interval and forward overlap.
Here is the calculations done to generate the speed of drone.
```
    forward_photo_height = agl * VERTICAL_FOV
    side_photo_width = agl * HORIZONTAL_FOV
    forward_overlap_distance = forward_photo_height * forward_overlap / 100
    side_overlap_distance = side_photo_width * side_overlap / 100
    forward_spacing = forward_photo_height - forward_overlap_distance
    side_spacing = side_photo_width - side_overlap_distance
    ground_speed = forward_spacing / image_interval
```

Do we restrict all of these dependent parameters ?

**Comment by ivangayton** [comment]
- Eventually we should probably put some suggested bounds on the parameters, yes, but for now I recommend we simply cap the speed at 12 m/s.

This will in some cases result in higher forward overlap than anticipated; the drone will be making photos at the same frequency as anticipated, but moving slower than anticipated. For now this is not the worst problem we could have; it may result in highee numbers of photos that need to be uploaded, take up more storage on the servers, etc., but it won't degrade the quality of the photogrammetric outputs (quite the contrary, higher overlap means better quality outputs).

The moment we provide a KML with a speed greater than 12 m/s, the controller will refuse to load it without automatically adjusting it, which makes it unusable. For now, I think we should just cap that output value.
